extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/QuteProcessor.java

```diff
@@ -55,6 +56,8 @@
 import org.jboss.jandex.TypeVariable;
 import org.jboss.logging.Logger;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
Pls do not use guava in this module. You can use the immutable map from JDK (`Map.of()`, `Map.copyOf()`, etc.).

Reply:
Sure, fixed, thanks